### PR TITLE
remove agenda and day view from calendar

### PR DESCRIPTION
**Pull request by matthewfollegot**
- [x] removed day and agenda option from calendar
- [x] swapped week with work-week view
- [x] made work-week view default

### CODE REVIEW

**Comment by matthewfollegot** [comment]
same as #186

File: src/pages/View/CalendarView/CalendarView.js [hunk 1]
```diff
@@ -42,17 +42,17 @@ export default class CalendarView extends Component {
             <div>
                 <ScrollArea>
                     <div className="box">
-                <BigCalendar
+                    <BigCalendar
```
Comment (Ruxuan) [inline src/pages/View/CalendarView/CalendarView.js]: Indent lines 45 - 56

File: .env [hunk 2]
```diff
@@ -1,3 +1,3 @@
 NODE_ENV="development"
-MONGODB_USERNAME="ASK GREG FOR THE MONGODB CREDENTIALS"
-MONGODB_PASSWORD="ASK GREG FOR THE MONGODB CREDENTAILS"
+MONGODB_USERNAME="matthewfollegot"
```
Comment (Ruxuan) [inline .env]: Remove .env change from commit

**Comment by matthewfollegot** [comment]
@Ruxuan thanks for the review, it should all be good now

**Review by Ruxuan — APPROVED** [review]

**Pull request merged**